contracts/utility/MetadataViews.cdc

```diff
@@ -1,4 +1,4 @@
-import FungibleToken from "./FungibleToken.cdc"
+import FungibleToken from "./utility/FungibleToken.cdc"
```

Comment:
This needs to not have `utility` in it, right?

Reply:
Yep right nice catch